### PR TITLE
Added ability to specify the number of upload attempts via command line.

### PR DESCRIPTION
**Pull request by nmoinvaz**

### CODE REVIEW

**Comment by nmoinvaz** [comment]
@thomasrockhu any idea of if this and #277 will be merged? Thanks in advance.

File: codecov/__init__.py [hunk 1]
```diff
@@ -1149,6 +1157,7 @@ def main(*argv, **kwargs):
                     res = retry_upload(
                         "%s/upload/v2?%s" % (codecov.url, urlargs),
                         requests.post,
+                        codecov.tries,
```
Comment (thomasrockhu) [inline codecov/__init__.py:1160]:
```suggestion
                        retries=codecov.tries,
```

File: codecov/__init__.py [hunk 2]
```diff
@@ -1128,6 +1135,7 @@ def main(*argv, **kwargs):
                         s3 = retry_upload(
                             upload_url,
                             requests.put,
+                            codecov.tries,
```
Comment (thomasrockhu) [inline codecov/__init__.py:1138]:
```suggestion
                            retries=codecov.tries,
```

File: codecov/__init__.py [hunk 3]
```diff
@@ -1108,6 +1114,7 @@ def main(*argv, **kwargs):
                     res = retry_upload(
                         "%s/upload/v4?%s" % (codecov.url, urlargs),
                         requests.post,
+                        codecov.tries,
```
Comment (thomasrockhu) [inline codecov/__init__.py:1117]:
```suggestion
                        retries=codecov.tries,
```

File: codecov/__init__.py [hunk 4]
```diff
@@ -407,6 +407,12 @@ def main(*argv, **kwargs):
         help="Specify a custom pr number, provided automatically for supported CI companies",
     )
     advanced.add_argument("--tag", default=None, help="Git tag")
+    advanced.add_argument(
+        "--tries",
+        default=3,
+        type=int,
+        help="Specify the number of attempts to make when uploading coverage report",
```
Comment (thomasrockhu) [inline codecov/__init__.py:414]:
```suggestion
        help="Specify the total number of attempts to make when uploading coverage report",
```

**Pull request merged**